cmd/kueueviz/frontend/src/CohortDetail.jsx

```diff
@@ -45,15 +45,18 @@ const CohortDetail = () => {
     );
   }
 
+  const clusterQueues = cohortDetails.clusterQueues || [];
+  const hasClusterQueues = Array.isArray(clusterQueues) && clusterQueues.length > 0;
```

Comment:
```suggestion
  const hasClusterQueues = clusterQueues.length > 0;
```
Is it possible something else?